lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.ts

```diff
@@ -192,6 +192,14 @@ export class CardViewTextItemComponent extends BaseCardView<CardViewTextItemMode
         }
     }
 
+    checkKeyboardEvent(event: any) {
```

Comment:
can you rename it to make that function more meaningful? It just says "check keyboard event" but it does not say what that function really does, that it "undos"

Reply:
hi alek, i have replaced with undoText()